### DISCUSSION TITLE
Rough diffuse BSDF

### DISCUSSION THREAD

**Discussion by G-nn-r** [mitsuba-renderer/mitsuba3#807] [Feature requests] [answered]
I was wandering why there is no support for a `roughdiffuse` material plugin. All other main materials (`glass`, `conductor`, `plastic`) have their rough counterpart. The Blender AddOn even explicitly states that Mitsuba currently does not support rough diffuse materials (see [here](https://github.com/mitsuba-renderer/mitsuba-blender/blob/master/mitsuba-blender/io/exporter/materials.py#L87)). 

Is it planned to add this as a feature? Or should just the [Principled BSDF](https://mitsuba.readthedocs.io/en/stable/src/generated/plugins_bsdfs.html#the-principled-bsdf-principled) be used instead?

**Comment by njroussel** [comment, marked as answer]
Hi @G-nn-r 

I don't know if there is a specific reason. It might simply never have been ported from Mitauba 0.6 because there never was a need for it. At this time, there is no plan to implement/add it ourselves. We'd welcome a PR for it.

I would indeed recommend using the `principled` BSDF for now.
